### PR TITLE
[hotfix] [docs] Refer to Running in an IDE section early on

### PR DESCRIPTION
**Pull request by honnix**
## What is the purpose of the change

When following this doc, I found myself searching on Google instead of reading the final section that is pretty further away. It might be useful to have an early reference.

## Brief change log

Refer to "Running in an IDE" section when we first mentioning running directly inside IDE.

## Verifying this change

This change is a trivial doc change without any test coverage.

## Does this pull request potentially affect one of the following parts:

NA.

## Documentation

NA.

### CODE REVIEW

File: docs/content/docs/try-flink/datastream.md [hunk 1]
```diff
@@ -902,4 +903,4 @@ You should see the following output in your task manager logs:
 
 Running the project in an IDE may result in a `java.lang.NoClassDefFoundError` exception. This is probably because you do not have all required Flink dependencies implicitly loaded into the classpath.
 
-* IntelliJ IDEA: Go to Run > Edit Configurations > Modify options > Select `include dependencies with "Provided" scope`. This run configuration will now include all required classes to run the application from within the IDE.
\ No newline at end of file
+* IntelliJ IDEA: Go to Run > Edit Configurations > Modify options > Select `include dependencies with "Provided" scope`. This run configuration will now include all required classes to run the application from within the IDE.
```
Comment (honnix) [inline docs/content/docs/try-flink/datastream.md:906]: I guess github gets confused. There is no change.

**Comment by MartijnVisser** [comment]
@slinkydeveloper What do you think?

**Comment by MartijnVisser** [comment]
@infoverload @slinkydeveloper What do you think?

File: docs/content/docs/try-flink/datastream.md [hunk 2]
```diff
@@ -120,6 +120,7 @@ You can edit the `groupId`, `artifactId` and `package` if you like. With the abo
 Maven will create a folder named `frauddetection` that contains a project with all the dependencies to complete this tutorial.
 After importing the project into your editor, you can find a file `FraudDetectionJob.java` (or `FraudDetectionJob.scala`) with the following code which you can run directly inside your IDE.
 Try setting break points through out the data stream and run the code in DEBUG mode to get a feeling for how everything works.
+Having trouble running directly inside your IDE? Please read the section [Running in an IDE](#running-in-an-ide).
```
Comment (slinkydeveloper) [inline docs/content/docs/try-flink/datastream.md:123]: I don't like the "Having trouble" wording. Maybe just "If you wanna run directly in you IDE, please read the section..."

Comment (honnix) [reply docs/content/docs/try-flink/datastream.md:123]: Sure.

File: docs/content/docs/try-flink/datastream.md [hunk 3]
```diff
@@ -120,6 +120,7 @@ You can edit the `groupId`, `artifactId` and `package` if you like. With the abo
 Maven will create a folder named `frauddetection` that contains a project with all the dependencies to complete this tutorial.
 After importing the project into your editor, you can find a file `FraudDetectionJob.java` (or `FraudDetectionJob.scala`) with the following code which you can run directly inside your IDE.
 Try setting break points through out the data stream and run the code in DEBUG mode to get a feeling for how everything works.
+If you want to run directly in you IDE, please read the section [Running in an IDE](#running-in-an-ide).
```
Comment (MartijnVisser) [inline docs/content/docs/try-flink/datastream.md:123]:
```suggestion
Please read the section [Running in an IDE](#running-in-an-ide) for all details on how to run Flink applications in your IDE.
```

**Review by MartijnVisser — CHANGES_REQUESTED** [review]
Thanks for the PR and the fixes. I've made a suggestion for the doc. 

What is more important, is that this PR is targeted towards the `release-1.14` branch. We should first target the `master` branch. 

It will also be important to add this English sentence to the Chinese documentation to avoid that it goes out of sink. That file can be found in `docs/content/docs/try-flink/datastream.md` and should be added to L136

**Comment by honnix** [comment]
@MartijnVisser Thanks for the suggestions.

I didn't realize it targets `release-1.14`. I will re-target it.

I can do the Chinese translation. Thanks for the reminder.

**Comment by honnix** [comment]
Looks like this has been fixed by https://github.com/apache/flink/commit/b800fb2767d570f566269aed9869af9d71c6adba already in main branch. I guess I reached to `release-1.14` branch by following the link in the rendered doc.

Closing this. Thanks for the help and sorry for the confusion.

**Comment by honnix** [comment]
I made the Chinese translation in https://github.com/apache/flink/pull/19044

**Pull request closed without merging**